model-mesh/samples/predictor_mlserver.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: serving.kserve.io/v1alpha1
+kind: Predictor
+metadata:
+  name: minimal-mlserver-predictor
+spec:
+  modelType:
+    name: mlserver
+  path: mnt/models
+  storage:
+    s3:
+      secretKey: localMinIO
+      bucket: triton-models # Change this to appropriately bucket name
```

Comment:
appropriate, not appropriately

Reply:
that's what I get for trusting the upstream file as-is